subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -133,6 +134,29 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
         "jobSlug" to "FunctionalTests"
     )
 
+    configurationsContainer.register("Local") {
```

Comment:
it will be better to add this config simultaneously with version bump probably

Reply:
removed